db/model/subject.js

```diff
@@ -440,9 +446,7 @@ module.exports = function subject(seq, dataTypes) {
 
               return resolve(inst);
             })
-            .catch((err) => {
-              return reject(err);
-            });
+            .catch(reject);
```

Comment:
There's a coveralls incident going on: https://github.com/lemurheavy/coveralls-public/issues/892

Let's not let ourselves get blocked by the coveralls calculation. Feel free to approve or reject PRs on their merit :)